components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/impl/ApplicationsApiServiceImpl.java

```diff
@@ -67,12 +67,20 @@ public class ApplicationsApiServiceImpl implements ApplicationsApiService {
     @Autowired
     private ServerApplicationSharingService applicationSharingService;
 
+    @Deprecated
     @Override
     public Response getAllApplications(Integer limit, Integer offset, String filter, String sortOrder, String sortBy,
                                        String requiredAttributes) {
 
-        ApplicationListResponse listResponse = applicationManagementService
-                .getAllApplications(limit, offset, filter, sortOrder, sortBy, requiredAttributes);
+        return getAllApplications(limit, offset, filter, sortOrder, sortBy, requiredAttributes, true);
+    }
+
+    @Override
+    public Response getAllApplications(Integer limit, Integer offset, String filter, String sortOrder, String sortBy,
+                                       String requiredAttributes, Boolean excludeSystemPortals) {
+
+        ApplicationListResponse listResponse = applicationManagementService.getAllApplications(limit, offset, filter,
+                sortOrder, sortBy, requiredAttributes, Boolean.parseBoolean(String.valueOf(excludeSystemPortals)));
```

Comment:
Can use Boolean.TRUE.equals(excludeSystemPortals)
```suggestion
                sortOrder, sortBy, requiredAttributes, Boolean.TRUE.equals(excludeSystemPortals));
```

Reply:
Addressed with https://github.com/wso2/identity-api-server/pull/711/commits/0e01f11829d4f0c5baf9cffd6a422a5e8bdb2150